src/settings.js

```diff
@@ -1135,6 +1135,10 @@ var MAIN_MODULE = 0;
 // [compile+link]
 var SIDE_MODULE = 0;
 
+// Include RPATH support.
+// [link]
+var SUPPORT_RPATH = 0;
```

Comment:
I don't think we should need a new setting for this.  Dynamic linking is pretty expensive thing to do use... i can't imagine the marginal cost of rpath is very much, is it?

Reply:
It doubles the size of the hello_dylink example. But I think in real cases the effect will be much smaller.

Reply:
Do you know where the size bloat is coming from?  Is the JS filesystem APIs?  

In any case i think we not add a new setting here.

Reply:
I'll remove it and rebaseline and we can see the added symbols. I suspect it's from including more of the file system.